src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -1433,21 +1433,20 @@ def value_counts(
         normalize: bool = False,
         sort: bool = True,
         ascending: bool = False,
-        bins: Any = None,
+        bins: int | None = None,
```

Comment:
```suggestion
        bins: Optional[int] = None,
```

Reply:
The linter changes it to `int | None` whenever I use `Optional`